boards/arduino-mega2560/src/lib.rs

```diff
@@ -22,7 +22,17 @@ pub use crate::hal::spi;
 pub use crate::hal::adc;
 
 pub type Delay = crate::hal::delay::Delay<hal::clock::MHz16>;
-pub type Serial<IMODE> = crate::hal::usart::Usart0<hal::clock::MHz16, IMODE>;
+
+pub mod usart {
+    pub use atmega2560_hal as hal;
```

Comment:
Not sure why, but I need to reimport.

Reply:
Should not be needed when you change the lines below like this:

```diff
-     pub type Usart0<IMODE> = crate::hal::usart::Usart0<hal::clock::MHz16, IMODE>;
+     pub type Usart0<IMODE> = crate::hal::usart::Usart0<crate::hal::clock::MHz16, IMODE>;
```